3529/2/number_checker.rb

```diff
@@ -0,0 +1,44 @@
+require 'yaml'
+require 'redis'
+require 'redis-activesupport'
+require 'redis-rails'
+require 'redis-namespace'
+require 'redis-rack-cache'
+
+class NumberChecker
+  def initialize(file_path)
+    @file_path = file_path
```

Comment:
If you need `file_path` only for load YML you don't need this instance var

Reply:
i need it for udating yaml file in update_file method here